source/learn/rosetta_stone.md

```diff
@@ -0,0 +1,1884 @@
+print *, shape(a)
+print *, size(a)
+print *, maxval(a)
+print *, minval(a)
+print *, sum(a)
+end program
+
+```
+:::::
+
+:::::{grid-item}
+```{code-block} Python
+:caption: Python
+
+from numpy import reshape
+a = reshape([1, 2, 3, 4, 5, 6], (2, 3))
+b = reshape([1, 2, 3, 4, 5, 6], (2, 3), order="F")
+print(a[0, :])
+print(a[1, :])
+print()
+print(b[0, :])
+print(b[1, :])
+
+```
+:::::
+
+:::::{grid-item}
+```{code-block} Fortran
+:caption: Fortran
+
+integer :: a(2, 3), b(2, 3)
+a = reshape([1, 2, 3, 4, 5, 6], [2, 3], order=[2, 1])
+b = reshape([1, 2, 3, 4, 5, 6], [2, 3])
+print *, a(1, :)
+print *, a(2, :)
+print *
+print *, b(1, :)
+print *, b(2, :)
+end program
+
+```
+:::::
+
+:::::{grid-item}
+```{code-block} text
+:caption: Python
+
+[1 2 3]
+[4 5 6]
+
+[1 3 5]
+[2 4 6]
+
+```
+:::::
+
+:::::{grid-item}
+```{code-block} text
+:caption: Fortran
+
+1           2           3
+4           5           6
+
+1           3           5
+2           4           6
+
+```
+:::::
+
+:::::{grid-item}
+```{code-block} Python
+:caption: Python
+
+from numpy import array, size, shape, max, min
+a = array([[1, 2, 3], [4, 5, 6]])
+print(shape(a))
+print(size(a, 0))
+print(size(a, 1))
+print(max(a))
+print(min(a))
+print(a[0, 0], a[0, 1], a[0, 2])
+print(a[1, 0], a[1, 1], a[1, 2])
+print(a)
+
+```
+:::::
+
+:::::{grid-item}
+```{code-block} Fortran
+:caption: Fortran
+
+integer :: a(2, 3)
+a = reshape([1, 2, 3, 4, 5, 6], [2, 3], order=[2, 1])
+print *, shape(a)
+print *, size(a, 1)
+print *, size(a, 2)
+print *, maxval(a)
+print *, minval(a)
+print *, a(1, 1), a(1, 2), a(1, 3)
+print *, a(2, 1), a(2, 2), a(2, 3)
+print "(3i5)", transpose(a)
+end program
+
+```
+:::::
+
+:::::{grid-item}
+```{code-block} text
+:caption: Python
+
+(2, 3)
+2
+3
+6
+1
+1 2 3
+4 5 6
+[[1 2 3]
+[4 5 6]]
+
+```
+:::::
+
+:::::{grid-item}
+   ```{code-block} text
+:caption: Fortran
+
+2 3
+2
+3
+6
+1
+1 2 3
+4 5 6
+1 2 3
+4 5 6
+
+```
+:::::
+
+:::::{grid-item}
+```{code-block} Python
+:caption: Python
+
+from numpy import array, all, any
+i = array([1, 2, 3])
+all(i == [1, 2, 3])
+any(i == [2, 2, 3])
+
+```
+:::::
+
+:::::{grid-item}
+```{code-block} Fortran
+:caption: Fortran
+
+integer :: i(3)
+i = [1, 2, 3]
+all(i == [1, 2, 3])
+any(i == [2, 2, 3])
+
+```
+:::::
+
+:::::{grid-item}
+```{code-block} Python
+:caption: Python
+
+from numpy import array, empty
+a = array([1, 2, 3, 4, 5, 6, 7, 8, 9, 10])
+b = empty(10)
+b[:] = 0
+b[a > 2] = 1
+b[a > 5] = a[a > 5] - 3
+
+```
+:::::
+
+:::::{grid-item}
+```{code-block} Fortran
+:caption: Fortran
+
+integer :: a(10), b(10)
+a = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+where (a > 5)
+    b = a - 3
+elsewhere (a > 2)
+    b = 1
+elsewhere
+    b = 0
+end where
+end program
+
+```
+:::::
+
+:::::{grid-item}
+```{code-block} Python
+:caption: Python
+
+from numpy import array, empty                     
+a = array([1, 2, 3, 4, 5, 6, 7, 8, 9, 10])     
+b = empty(10)                                  
+for i in range(len(a)):                 
```

Comment:
Please remove one space in all the lines of the block inside the `for` .

Reply:
Sure, @vmagnin